app/_src/gateway/kong-manager/auth/workspaces-and-teams.md

```diff
@@ -41,29 +41,30 @@ For this example, start by creating a simple workspace called `SecureWorkspace`.
 2. On the workspaces tab, click on **New Workspace**.
 3. Create a workspace named `SecureWorkspace` and select a color or image for the workspace avatar.
 
-    Each workspace name should be unique, regardless of letter case. For example, naming one workspace “Payments” and another one “payments” will create two different workspaces that appear identical.
-
     {:.important}
-    > Do not give a workspace the same name as any of these major routes in Kong Manager:
+    > Workspace names are **case sensitive** ("Payments" and "payments" are not equal), so it is recommended you give your worksapces unique names regardless of letter case to prevent confusion.
+    >
+    > <br>
+    > Additionally, do not give a workspace the same name as any of these major routes in Kong Manager:
     >
     |---------|-----------|--------------|---------------|
     | Admins  | APIs      | Certificates | Consumers     |
     | Plugins | Portal    | Routes       | Services      |
     | SNIs    | Upstreams | Vitals       | PermalinkStep |
 
-4. Click **Create New Workspace**.
-5. In the new workspace, click **Teams**.
-6. From the Teams page, click the **Roles** tab to view the default roles that come with {{site.base_gateway}}.
-7. Select SecureWorkspace to see its assigned roles.
+4. Click **Create New Workspace**, and you will see your new workspace dashboard.
+5. Click the **Teams** tab.
+6. From the Teams page, click the **Roles** tab.
+7. Select `SecureWorkspace` to view the default roles that come with {{site.base_gateway}}.
 
     By default, each new workspace has the following roles and privileges:
 
     | Role                     | Description                                                                                  |
     |--------------------------|----------------------------------------------------------------------------------------------|
-    | *workspace-admin*        | Can administer the objects in a workspace but can’t add new administrators to the workspace. |
-    | *workspace-portal-admin* | Can manage the Dev Portal. |
-    | *workspace-read-only*    | Can view anything in the workspace, but can’t make any changes. |
-    | *workspace-super-admin*  | Can do anything inside the workspace. |
+    | *workspace-admin*        | Full access to all endpoints in the workspace — except RBAC Admin API. |
```

Comment:
```suggestion
    | *workspace-admin*        | Full access to all endpoints in the workspace except the RBAC Admin API. |
```

Reply:
Totes, just an FYI I copied and pasted directly from the product for these 🤷

Reply:
😭